composer.json

```diff
@@ -1,47 +1,46 @@
 {
-  "name": "atk4/chart",
-  "type": "library",
-  "description": "ChartJS for Agile UI",
-  "keywords": [
-    "agile",
-    "data",
-    "framework",
-    "chart",
-    "chartjs"
-  ],
-  "homepage": "https://github.com/atk4/chart",
-  "license": "MIT",
-  "authors": [
-    {
-      "name": "Romans Malinovskis",
-      "email": "romans@agiletoolkit.org",
-      "homepage": "https://nearly.guru/"
-    }
-  ],
-  "minimum-stability": "dev",
-  "prefer-stable": true,
-  "config": {
-    "sort-packages": true
-  },
-  "require": {
-    "atk4/ui": "dev-develop"
-  },
-  "require-release": {
-    "atk4/ui": "~2.3.0"
-  },
-  "require-dev": {
-    "friendsofphp/php-cs-fixer": "^2.16",
-    "phpunit/phpcov": "*",
-    "phpunit/phpunit": "*"
-  },
-  "autoload": {
-    "psr-4": {
-      "Atk4\\Chart\\": "src/"
-    }
-  },
-  "autoload-dev": {
-    "psr-4": {
-      "Atk4\\Chart\\Tests\\": "tests/"
-    }
-  }
-}
+	"name" : "atk4/chart",
+	"type" : "library",
+	"description" : "ChartJS for Agile UI",
+	"keywords" : [
+		"agile",
+		"data",
+		"framework",
+		"chart",
+		"chartjs"
+	],
+	"homepage" : "https://github.com/atk4/chart",
+	"license" : "MIT",
+	"authors" : [{
+			"name" : "Romans Malinovskis",
+			"email" : "romans@agiletoolkit.org",
+			"homepage" : "https://nearly.guru/"
+		}
+	],
+	"minimum-stability" : "dev",
+	"prefer-stable" : true,
+	"config" : {
+		"sort-packages" : true
+	},
+	"require" : {
+		"atk4/ui" : "dev-develop"
+	},
+	"require-release" : {
+		"atk4/ui" : "~2.3.0"
+	},
+	"require-dev" : {
+		"friendsofphp/php-cs-fixer" : "^2.16",
+		"phpunit/phpcov" : "*",
+		"phpunit/phpunit" : "*"
+	},
+	"autoload" : {
+		"psr-4" : {
+			"Atk4\\Chart\\" : "src/"
+		}
+	},
+	"autoload-dev" : {
+		"psr-4" : {
+			"Atk4\\Chart\\Tests\\" : "tests/"
+		}
+	}
+}
```

Comment:
fix this EOL and remove unrelated "stacks changes" (should be only in https://github.com/atk4/chart/pull/16)

then we can merge into 3.0 and start 3.1